datalake/src/test/java/com/sequenceiq/datalake/service/sdx/SdxUpgradeServiceTest.java

```diff
@@ -55,7 +56,7 @@ public void setUp() {
     public void testUpdateRuntimeVersionFromCloudbreak() {
         when(sdxService.getById(1L)).thenReturn(sdxCluster);
         StackV4Response stackV4Response = getStackV4Response();
-        when(stackV4Endpoint.get(eq(0L), eq("test-sdx-cluster"), eq(Set.of()), anyString()))
```

Comment:
Please use `eq(true)` or `eq(false)` instead of `anyBoolean()` since this is important here.